### PR TITLE
Add read-write splitting tests for standard databases

### PR DESCRIPTION
**Pull request by aaron-congo**
- add read-write splitting tests for standard databases
- correct misnamed tables in Aurora integration tests
- ensure Aurora integration test tables are cleaned up

### CODE REVIEW

**Review by joyc-bq — APPROVED** [review]
looks good to me

File: wrapper/src/test/java/integration/host/StandardMysqlContainerTest.java [hunk 1]
```diff
@@ -32,7 +35,11 @@
 public class StandardMysqlContainerTest {
 
   private static final String STANDARD_TEST_RUNNER_NAME = "test-container";
-  private static final String STANDARD_MYSQL_HOST = "standard-mysql-container";
+  private static final String STANDARD_MYSQL_WRITER = "standard-mysql-writer.cluster-XYZ.us-east-2.rds.amazonaws.com";
```
Comment (sergiyvamz) [inline wrapper/src/test/java/integration/host/StandardMysqlContainerTest.java:38]: Do we need to use Amazon domain name style? Will "standard-mysql-writer" be enough?

File: wrapper/src/test/java/integration/host/StandardPostgresContainerTest.java [hunk 2]
```diff
@@ -32,7 +35,12 @@
 public class StandardPostgresContainerTest {
 
   private static final String STANDARD_POSTGRES_TEST_RUNNER_NAME = "test-container";
-  private static final String STANDARD_POSTGRES_HOST = "standard-postgres-container";
+  private static final String STANDARD_POSTGRES_WRITER =
```
Comment (sergiyvamz) [inline wrapper/src/test/java/integration/host/StandardPostgresContainerTest.java:38]: Similar question as above

File: wrapper/src/test/java/integration/host/StandardMysqlContainerTest.java (hunk 1 above)
Comment (aaron-congo) [reply wrapper/src/test/java/integration/host/StandardMysqlContainerTest.java:38]: Yeah just realized it should just be standard-mysql-writer, I'm working on changing that

File: wrapper/src/main/java/software/amazon/jdbc/util/ConnectionUrlParser.java [hunk 3]
```diff
@@ -38,24 +38,30 @@ public class ConnectionUrlParser {
...
-          });
+      final String[] hostArray = hosts.split(HOSTS_SEPARATOR);
+      for (int i = 0; i < hostArray.length; i++) {
+        final HostSpec host;
+        if (singleWriterConnectionString) {
+          final HostRole role = i > 0 ? HostRole.READER : HostRole.WRITER;
+          host = parseHostPortPair(hostArray[i], role);
+        } else {
+          host = parseHostPortPair(hostArray[i]);
+        }
```
Comment (sergiyvamz) [inline wrapper/src/main/java/software/amazon/jdbc/util/ConnectionUrlParser.java:59]: nit: Probably you can do it in a single line.

**Review by sergiyvamz — APPROVED** [review]

**Pull request merged**